loopdb/sqlc/migrations/000008_static_address_deposits.up.sql

```diff
@@ -24,7 +24,11 @@ CREATE TABLE IF NOT EXISTS deposits (
     timeout_sweep_pk_script BYTEA NOT NULL,
 
     -- expiry_sweep_txid is the transaction id of the expiry sweep.
-    expiry_sweep_txid BLOB
+    expiry_sweep_txid BLOB,
+
+    -- withdrawal_sweep_pk_script is the address that will be used to sweep the
+    -- deposit cooperatively with the server before it has expired.
```

Comment:
> before it has expired

Is it possible to withdraw after the deposit has expired?

Reply:
The client currently prevents selecting deposits for withdrawal that are in the expired state.

Reply:
> But that is useless to the client.

Wouldn't it save some sats for the client? IIUC, this is a way to spend the UTXO using MuSig2 instead of using tapscript (i.e. cheaper and more private).